spec/cases/each_with_ar_postgres.rb

```diff
@@ -0,0 +1,51 @@
+require './spec/cases/helper'
```

Comment:
since the code should be the same very time, this can be 1 file with a case statement for the setup